src/models/Wallet.ts

```diff
@@ -48,4 +57,18 @@ export default class Wallet implements ISigner {
       throw err;
     }
   }
+
+  /**
+   * We just need to pass a valid sendTo address belonging to that network for the
+   * qtumjs-wallet library to calculate the maxQtumSend amount.  It does not matter what
+   * the specific address is, as that does not affect the value of the
+   * maxQtumSend amount
+   */
+  private maxQtumSendToAddress = (networkName: string) => {
+    if (networkName === NETWORK_NAMES.MAINNET) {
+      return 'QN8HYBmMxVyf7MQaDvBNtneBN8np5dZwoW';
+    }
+    // TestNet or RegTest
+    return 'qLJsx41F8Uv1KFF3RbrZfdLnyWQzvPdeF9';
```

Comment:
can 1-liner this: `return networkName === NETWORK_NAMES.MAINNET ? 'QN8HYBmMxVyf7MQaDvBNtneBN8np5dZwoW' : 'qLJsx41F8Uv1KFF3RbrZfdLnyWQzvPdeF9';`